sdk/operationalinsights/Microsoft.Azure.OperationalInsights/src/Customized/Models/QueryResponseInnerError.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace Microsoft.Azure.OperationalInsights.Models
+{
+    public class QueryResponseInnerError
+    {
+        /// <summary>
+        /// A machine readable severity code.
+        /// </summary>
+        public int Severity { get; set; }
+        /// <summary>
+        /// A human readable severity message.
+        /// </summary>
+        public string SeverityName { get; set; }
+        /// <summary>
+        /// A human readable error message.
+        /// </summary>
+        public string Message { get; set; }
+        /// <summary>
+        /// A machine readable error code.
+        /// </summary>
+        public string Code { get; set; }
```

Comment:
should it be a string or int? how it is returned by Draft?

Reply:
_Severity_ and _SeverityName_ cannot be found in the Swagger file (05-19_Preview) so I chose Severity to be an integer and SeverityName to be a string.
_Code_ is a string by the [05-19_Preview Swagger file](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/operationalinsights/data-plane/Microsoft.OperationalInsights/preview/2021-05-19_Preview/OperationalInsights.json#L1277).